src/main/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -263,6 +265,14 @@ private String getRootUrl() {
 				"" : "/medic/_design/medic/_rewrite/");
 	}
 
+	private void browse(Uri url) {
+		if (url != null) {
+			container.load(url.toString(), null);
+		} else {
+			browseToRoot();
+		}
+	}
```

Comment:
I think we can get slightly better reuse here, for example, have a `getUrlToLoad` function which returns `url.toString()` or `getRootUrl`, and then have a `browseTo(url)` function which does the logging and calls `container.load`. What do you think?

Reply:
Done.